test/unit/fixture/reverse_with_layer_filtering_non_coarse_subset.js

```diff
@@ -0,0 +1,41 @@
+var vs = require('../../../query/reverse_defaults');
+
+module.exports = {
+  'query': {
+    'bool': {
+      'filter': [
+        {
+          'geo_distance': {
+            'distance': '500km',
```

Comment:
Oh, yes i guess it's actually line 161 that came from 141, which is probably a very old line that dates back to the days when we actually did query in a 500km radius. I think it's important to remove but I'll do it in a separate PR so not to distract from here.